index.md

```diff
@@ -39,9 +39,22 @@ EDT | CET | Speaker | Presentation Title
 14:45 | 20:45 |  [Jérôme Dockès](https://jeromedockes.github.io/) | Continuous integration for academic software
 15:30 | 21:30 |  [Alejandro de la Vega](https://adelavega.github.io/)  | The Python ecosystem for neuroimaging
 
+## Scientific day (22 May): Confirmed Speakers
+
+Speaker | Presentation Title
+--------|--------------------
+[Carsen Stringer](http://www.gatsby.ucl.ac.uk/~cstringer/)  |  Rastermap: A Visualization Tool for High-dimensional Neural Data
+[Eva Dyer](https://dyerlab.gatech.edu/people/pi-profile/)  | Comparing High-Dimensional Neural Recordings Across Time, Space, and Behavior
+[Jo Etzel](https://sites.wustl.edu/ccplab/people/jo-etzel/) | Pattern Similarity Analyses of Frontoparietal Task Coding: Individual Variation and Genetic Influences
+[Luke J Chang](https://pbs.dartmouth.edu/people/luke-j-chang-0)  | *to be announced*
+[Aki Nikolaidis](https://childmind.org/bio/aki-nikolaidis-phd/)  | Bagging Improves Reproducibility of Functional Parcellation of the Human Brain
+[Sylvia Villeneuve](http://www.villeneuvelab.com/en/home/)/[Alexa Pichet Binette](https://www.pubfacts.com/author/Alexa+Pichet-Binette) | Predicting Functional Brain Aging in Preclinical Alzheimer’s Disease
+
 ## Sprint Registration
 
-If you would like to participate in the 3 day coding sprint, please fill the form here:
+If you would like to participate in the 2 day coding sprint, please fill the
+form below. This will help us select issues to be addressed during the sprint
+and organize the participation of nilearn maintainers.
```

Comment:
I'm not sure how this would help us select issues ? Since we don't currently ask for information on their background or interests.

Regardless, I think we could add information on communicating the structure of the sprint (as we finalize it) :point_down: 

```suggestion
If you would like to participate in the 2 day coding sprint, please fill the
form below. This will help us select issues to be addressed during the sprint
and organize the participation of nilearn maintainers. It will also allow us to more
directly provide you information on how the online-only sprint will be structured.
```

Reply:
I think all information should be provided through the event page so that it's all in one place and people who don't register get it too.
Re. the motivation for asking to register, I'm happy to change it to anything else. 
BTW do we need to know more or less how many people will listen to the talks (for the other 2 days) to choose the video tool we will use?

Reply:
> I think all information should be provided through the event page so that it's all in one place and people who don't register get it too.

Yes, absolutely. I mentioned this out-of-band (on Slack, sorry), but I think we should both email and post on the website. In my experience, folks usually like having that direction communication !

> BTW do we need to know more or less how many people will listen to the talks (for the other 2 days) to choose the video tool we will use?

I think there's no good way to handle that -- even if folks register ahead of time, day-of it's likely that some folks will drop and (many ?) others will want to join. Unless we're restricting the stream and asking folks to eg enter a password, I'd argue we should plan for a high number.